diagnostic.md

```diff
@@ -177,5 +201,6 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+Will use the @media (min-width) 400px 25% since the @media (min-width) for 30% is 800px
+so 412.5px
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px.